expect.go

```diff
@@ -213,20 +221,16 @@ type Reporter interface {
 	Errorf(message string, args ...interface{})
 }
 
-// LoggerReporter combines Logger and Reporter interfaces.
+// LoggerReporter combines Logger and Reporter interfaces, and adds the Name()
+// method requirement in order to provide the name of the current test.
+//
+// Note: all methods are provided when using a *testing.T struct.
 type LoggerReporter interface {
+	Name() string
 	Logger
 	Reporter
 }
```

Comment:
> Please let me know if you would like to leave some issues unresolved in this PR and postpone them to future PR(s). It's ok since we're merging into a feature branch.

since you're the reviewer here, your choice, i'm fine with either :)

Reply:
addressed